README.md

```diff
@@ -76,24 +76,24 @@ cosign verify-blob --key sbom/key.pub --signature sbom/kots-sbom.tgz.sig sbom/ko
 
 ### How To
 
-#### Deploying an application to a different namespace from Kots Admin
+#### Deploying an application to a different namespace from KOTS Admin
 
 If you need to test deploying an application to a different namespace, you'll need to first create the additional namespace in Okteto.
 Your permissions will be the same between both namespaces, and you will be able to create deploy/resources there.
 
-##### Use the Kots CLI while Kots Admin is running
+##### Use the KOTS CLI while KOTS Admin is running
 
 1. `okteto up` - Put the the kots pod into dev mode
```

Comment:
just a nitpick: this one could be capitalized too: "Put the kots pod into dev mode"